pipeline/src/interface/ArgInterface.py

```diff
@@ -45,7 +44,7 @@ def _service_date(self, arg):
             return datetime.strptime(arg, "%Y-%m-%d")
         except ValueError:
             err_msg = "Invalid service date format: {0}, YYYY-MM-DD expected.".format(arg)
-            # TODO uncomment when ready to enable logging output
```

Comment:
Are you intending to leave these as comments for now?

Reply:
That is part of my issue this sprint, I'll be taking care of it.

Reply:
Gotcha, alright